public/locales/ko-KR/translations.json

```diff
@@ -339,6 +340,14 @@
   "unauthorize": "인증 취소",
   "missed_validations": "{{count}}회의 유효성 검증이 누락되었습니다",
   "incomplete": "미완료",
+  "base_fee": null,
+  "account_reserve": null,
+  "object_reserve": null,
+  "amendments": null,
+  "amendment_name": null,
+  "amendment_id": null,
+  "vote": null,
+  "no_amendment_in_voting": null,
```

Comment:
why are all those null?

Reply:
The translation for languages other than English would be added later by translators. By giving them `null` value for now, it would default to English until we have the translation